README.md

```diff
@@ -380,7 +402,7 @@ method takes two arguments:
 ### Profunctor
 
 A value that implements the Profunctor specification must also implement
-the Functor specification.
+the Functor and Contravariant Functor specifications.
```

Comment:
Hmm.

Am I write in thinking that there could be a class `AmateurFunctor` with the opposite problem?

```hs
class AmateurFunctor f where
  amateurmap :: (a -> b) -> (c -> d) -> f a d -> f b c
```

I'm slightly unclear if there's a connection between the variance of the type arguments in the kind and the variance of type parameters in the type.